src/function-builder.ts

```diff
@@ -139,12 +139,12 @@ export function region(
 /**
  * Configure runtime options for the function.
  * @param runtimeOptions Object with three optional fields:
- * 1. failurePolicy: failure policy policy of the function, boolean `true` is
- *    equivalent to providing an empty policy.
- * 2. memory: amount of memory to allocate to the function, possible values
- *    are: '128MB', '256MB', '512MB', '1GB', and '2GB'.
- * 3. timeoutSeconds: timeout for the function in seconds, possible values are
- *    0 to 540.
+ * 1. failurePolicy: failure policy of the function, with boolean true being
+ *    equivalent to providing an empty retry object.
+ * 2. memory: amount of memory to allocate to the function, with possible
+ *    values being '128MB', '256MB', '512MB', '1GB', and '2GB'.
+ * 3. timeoutSeconds: timeout for the function in seconds, with possible
+ *    values being 0 to 540.
```

Comment:
Can we add this to the end of this docstring:

```
*
* Value must not be null.
*/
```
This will soon be exposed automatically via TypeDoc so I was thinking we can make this look closer to the reference here https://firebase.google.com/docs/reference/functions/functions.FunctionBuilder.html#runWith.

Reply:
668c02fe4b9fbff28136c79e36959c1df4d23e33